manifests/foobar2000/foobar2000/1.5.4.0.yaml

```diff
@@ -0,0 +1,13 @@
+Id: foobar2000.foobar2000
+Publisher: foobar2000
+Name: foobar2000
+Version: 1.5.4.0
+License: BSD
```

Comment:
I'm fairly sure the foobar2000 app is not BSD licensed, the official website lists it as [foobar2000 license](https://www.foobar2000.org/license) which is a custom freeware thing.

Reply:
what should I change it to? `Copyright © Peter Pawlowski` or `Freeware` ?

Reply:
Copyright would be more accurate I think, you can also add the additional field LicenseUrl that points to the document. :)